integration_tests/snapshot_test.go

```diff
@@ -336,3 +337,53 @@ func (s *testSnapshotSuite) TestSnapshotRuntimeStats() {
 		"block: {cache_hit_count: 20, read_count: 40, read_byte: 30 Bytes}}}"
 	s.Equal(expect, snapshot.FormatStats())
 }
+
+func (s *testSnapshotSuite) TestRCRead() {
+	for _, rowNum := range s.rowNums {
+		s.T().Logf("test BatchGet, length=%v", rowNum)
+		txn := s.beginTxn()
+		for i := 0; i < rowNum; i++ {
+			k := encodeKey(s.prefix, s08d("key", i))
+			err := txn.Set(k, valueBytes(i))
+			s.Nil(err)
+		}
```

Comment:
What about move `keys := makeKeys(rowNum, s.prefix)` here instead of copying the implementation again?